webservices/common/models.py

```diff
@@ -855,6 +856,17 @@ class Filings(db.Model):
     update_date = db.Column(db.Date)
 
     @property
+    def document_description(self):
+        if self.report_type_full and self.report_type_full != '':
```

Comment:
We don't need to check both `self.report_type_full` and `self.report_type full != ''`, since the empty string is falsy in Python. We can just say `if self.report_type_full:` here.